CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change log
 
 ## [v0.10.0] - unreleased
+* Fix cursor hiding in threaded run
```

Comment:
Any fixes should go under the `## Fixed` section, which follows last. I'd consider renaming since we're not trying to hide but actually show the cursor on failure. I suggest `Fix automatic spinning to show cursor on error`.

Reply:
But this is not the fix. Showing on error is already done, but it was done in a way that breaks cursor hiding.